### PR TITLE
Add 'pull handbrake' to vehicle action menu

### PR DESCRIPTION
**Pull request by Pupsi-Mupsi**
#### Summary

`SUMMARY: Interface "Add pull handbrake to vehicle action menu"`

#### Purpose of change

This provides an additional way for pulling the handbrake.
Might especially help new players.
@mlangsdorf - Could you take a look at it? Is there a better way to implement this?

---

#### Screenshot(s)

After:

grafik

### CODE REVIEW

File: src/vehicle_use.cpp [hunk 1]
```diff
@@ -797,6 +797,32 @@ void vehicle::use_controls( const tripoint &pos )
     }
 }
 
+static void handbrake()
```
Comment (Qrox) [inline src/vehicle_use.cpp:800]: You should make `handbrake` an exported function instead of copying it around.

Comment (Pupsi-Mupsi) [reply src/vehicle_use.cpp]: Thanks. Like this?

File: src/handle_action.cpp [hunk 2]
```diff
@@ -137,6 +137,8 @@ extern bool add_best_key_for_action_to_quick_shortcuts( action_id action,
 extern bool add_key_to_quick_shortcuts( int key, const std::string &category, bool back );
 #endif
 
+void handbrake();
```
Comment (Qrox) [inline src/handle_action.cpp:140]: Please move this to `handle_action.h`

File: src/vehicle_use.cpp [hunk 3]
```diff
@@ -55,6 +55,7 @@
 #include "vpart_position.h"
 #include "vpart_range.h"
 #include "weather.h"
+#include "handle_action.cpp"
```
Comment (Qrox) [inline src/vehicle_use.cpp:58]: And include `handle_action.h` here instead.

Comment (Pupsi-Mupsi) [reply src/vehicle_use.cpp]: Thanks!

File: src/handle_action.h [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#pragma once
+#ifndef CATA_SRC_HANDLE_ACTION_H
+#define CATA_SRC_HANDLE_ACTION_H
+
+void handbrake();
```
Comment (Qrox) [inline src/handle_action.h:5]: Oops, didn't know that `handle_action.h` didn't exist. Based on what `handbrake` does it's probably better to move this to `vehicle.h` and move the definition of `handbrake` to `vehicle_use.cpp`?

Comment (Pupsi-Mupsi) [reply src/handle_action.h:5]: ;-) Sure. Thanks for taking the time to help me with that. I'll see what I can do.

**Pull request merged**